Automattic/jetpack
# Update i18n-calypso

Update `i18n-calypso` dependency which in turn is mainly update of its dependencies. As you can see in the lockfile, it should get rid of some duplicate dependencies.

[See i18n-calypso changelog](https://github.com/Automattic/wp-calypso/blob/9ebe145354a7e7ab5540ce2ecd861c61ff7f963b/packages/i18n-calypso/CHANGELOG.md)

#### Changes proposed in this Pull Request:
* No functional changes.

#### Testing instructions:

- Confirm that everything with translations works in Jetpack dash normally.
- Confirm that `yarn build-languages` builds and updates `_inc/jetpack-strings.php` as well as languages json files.

#### Proposed changelog entry for your changes:

- N/A
